app/config/settings_dev.py

```diff
@@ -20,6 +20,8 @@
 
 ALLOWED_HOSTS = ['*']
 
+SECRET_KEY = 'fJZUSfuyvLGR5roQklcqBrvc/ceFPk8dcePzZIVqOKc='
```

Comment:
Why overriding the SECRET_KEY here ? It should already be set in `.env.local` from `.env.default` or in `docker-compose-ci.yml`.